ponyo/generate_template_data.py

```diff
@@ -0,0 +1,185 @@
+'''
+Author: Alexandra Lee
+Date Created: 15 June 2020
+
+Script to generate gene expression compendia using experiment-preserving approach using a specific template experiment.
+This script is used to generate a null set of gene expression data with some context, defined by the template experiment
+'''
+
+import os
+import ast
+import pandas as pd
+import numpy as np
+import random
+import glob
+import pickle
+from keras.models import load_model
+from sklearn import preprocessing
+
+from ponyo import generate_labeled_data
```

Comment:
Wouldn't `import generate_labeled_data` work?

Reply:
Technically i believe I can, this was just to be explicit